src/Twig/Extension/BreadcrumbsExtension.php

```diff
@@ -78,8 +94,14 @@ public function renderBreadcrumbsForTitle(
         AdminInterface $admin,
         string $action
     ): string {
-        return $environment->render('@SonataAdmin/Breadcrumb/breadcrumb_title.html.twig', [
-            'items' => $this->breadcrumbsBuilder->getBreadcrumbs($admin, $action),
-        ]);
+        @trigger_error(sprintf(
+            'The method "%s()" is deprecated since sonata-project/admin-bundle 4.x and will be removed in 5.0.'
+            .' Use "%s::%s() instead.',
```

Comment:
Please, be aware about this suggestion in the remaining occurrences (I think there are other 14 cases like this in the PR).
```suggestion
            .' Use "%s::%s()" instead.',
```

Reply:
Sorry, I fixed all of them